core/src/trezor/ui/layouts/mercury/__init__.py

```diff
@@ -316,6 +314,29 @@ def show_warning(
     )
 
 
+def show_danger(
+    br_name: str,
+    content: str,
+    content_short: str | None = None,
```

Comment:
I don't think the distinction between `content`/`content_short` is a good idea. I think we can keep the `confirm_ethereum_unknown_contract_warning` function which calls the `show_danger` with the appropriate string.

Reply:
Fixed!